pages/selectable-item/virtual-scroll.page.tsx

```diff
@@ -0,0 +1,98 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { useState } from 'react';
+
+import { Multiselect, MultiselectProps, SpaceBetween } from '~components';
+import Autosuggest, { AutosuggestProps } from '~components/autosuggest';
+import Select, { SelectProps } from '~components/select';
+
+import { SimplePage } from '../app/templates';
+
+const options: SelectProps.Options = Array.from({ length: 1000 }, (_, i) => ({
+  value: `${i}`,
+  label: `Option ${i + 1}`,
+}));
+
+const autosuggestOptions: AutosuggestProps.Options = Array.from({ length: 1000 }, (_, i) => ({
+  value: `Option ${i + 1}`,
+  description: `Description for option ${i + 1}`,
+}));
+
+export default function () {
+  const [selected, setSelected] = useState<SelectProps['selectedOption']>(null);
+  const [selectedMulti, setSelectedMulti] = useState<MultiselectProps.Options>(options.slice(0, 2));
+  const [selectedMultiWithSelectAll, setSelectedMultiWithSelectAll] = useState<MultiselectProps.Options>([]);
+  const [autosuggestValue, setAutosuggestValue] = useState('');
+
+  return (
+    <SimplePage title="Virtual Scroll" i18n={{}}>
```

Comment:
If you need screenshot tests for this page, you need to also add `screenshotArea={{}}` here